bin/terminate_build

```diff
@@ -21,6 +22,7 @@ def execute_system_command(command)
 end
 
 CDO.log.info 'Terminating currently executing build (aws/ci_build) including its child processes ...'
+ChatClient.log "A user has invoked bin/terminate_build on this environment's daemon server ...", color: 'red'
```

Comment:
Oh, I didn't. I was just saying that I'd like to see these `log` statements moved below the `raise` statements below, so they only output if we're actually going to attempt to terminate the build

Reply:
Done!